packages/styled-components/components/image/index.js

```diff
@@ -36,16 +36,16 @@ const Image = (props) => {
     className,
     credit,
     fallbackSrc,
+    height,
     loading,
     objectFit,
-    // eslint-disable-next-line no-unused-vars
```

Comment:
I went ahead and removed these since they're currently unused and I think we can better implement them elsewhere.